### DISCUSSION TITLE
Welcome to rust-skia Discussions!

### DISCUSSION THREAD

**Discussion by Brooooooklyn** [rust-skia/rust-skia#434] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ctrlcctrlv** [comment]
Huh what is this? Are you maintainer now what's happening

**Reply by Brooooooklyn** [reply]
Haha, I created `rust-skia ` 2 years ago, now @pragmatrix mainly maintain this organization, and I am still following the development of `rust-skia`.
